src/classes/music.rb

```diff
@@ -0,0 +1,14 @@
+class MusicAlbum < Item
+  attr_reader :spotify
+
+  def initialize(spotify, id, date)
+    super(id, date)
+    @spotify = spotify
```

Comment:
The initialize method is correct :smile: but it would be better if we stick with the properties' names [ provided in the project requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/images/catalog_of_my_things.png?raw=true)
So instead of spotify, it's better to use `on_spotify`, and instead of date, it's better to use `publish_date` :+1:

Reply:
That's right. We need to maintain consistency